source/panel_finder/panel_finder.py

```diff
@@ -17,12 +16,21 @@ def combined_panel(rect_a, rect_b):
 
 
 class PanelFinder(Module):
-    def __init__(self, state=None):
+    def __init__(self, state):
+        if state == "tensorflow":
+            from source.panel_finder.panel_classifier.panel_classifier import PanelClassifier
+            state = {"mode": "load"}
```

Comment:
This is dangerous because it overrides all other values in `state` (not that there are any yet).

When panel_finder gets used as a module of a larger component of this project, I think this may end up overriding some state values undesirably.

Reply:
Yea that's not good. Changed it to state["mode"] = "load"